README.md

```diff
@@ -2,7 +2,7 @@
 
 A Ruby library to expand shortened URLs.
 
-**Current version:** 1.2.3  
+**Current version:** 1.2.4
```

Comment:
Whitespace restored. Also, `{  "trim_trailing_white_space_on_save": false }` added to my local `Markdown.sublime-settings` for the future…